frontend/src/services/locations-services.js

```diff
@@ -0,0 +1,20 @@
+export const mangelLocations = [
```

Comment:
This is just some mock data, that is going to be removed?

Reply:
yes, just temporary, until I get data from backend